addons/crm/models/crm_lead_scoring_frequency.py

```diff
@@ -28,3 +34,55 @@ def _get_default_color(self):
         ondelete='cascade',
     )
     color = fields.Integer('Color', default=_get_default_color)
+
+
+class CrmLeadScoringPendingUpdate(models.Model):
+    _name = 'crm.lead.scoring.pending.update'
+    _description = 'Lead Scoring Pending Update'
+    _order = 'create_date ASC, id'
+
+    lead_id = fields.Many2one('crm.lead', required=True, ondelete="cascade")
+    from_lost = fields.Boolean(required=True)
+    from_won = fields.Boolean(required=True)
+    to_lost = fields.Boolean(required=True)
+    to_won = fields.Boolean(required=True)
+
+    def _cron_process_pls_pending_updates(self):
```

Comment:
Shouldn't we move this in the crm.lead model and change accordingly the cron ? We don't really use anything from the model here